packages/stark-ui/src/modules/generic-search/components/generic-search/generic-search.component.ts

```diff
@@ -235,8 +237,9 @@ export class StarkGenericSearchComponent extends AbstractStarkUiComponent implem
 	 * Component lifecycle hook
 	 */
 	public ngAfterContentInit(): void {
-		if (typeof this.searchFormComponent !== "undefined") {
-			this.genericForm = this.searchFormComponent.searchForm;
+		// TODO write a test for this
+		if (!this.searchFormComponent) {
+			throw new Error("#searchFormComponent content-child should be added to stark-generic-search");
```

Comment:
Could you add a test for this? And also, could you adapt the error message to follow the same format as the other errors: `StarkXXComponent: the error message`